### PR TITLE
Promise. Add `sequenceP`.

### PR DESCRIPTION
**Pull request by wowofbob**
`parallelP` analogue which computes supplied promises sequentially

### CODE REVIEW

**Comment by alstrup** [comment]
@dmitrybure LGTM, but you wrote promises, so worth giving you a vote on it.

File: lib/promise.flow [hunk 1]
```diff
@@ -79,6 +79,11 @@ export {
 
 	//mapP and mapPparallel were moved to arrayutils.flow
 
+	// This function takes an array of promises and ties them together
+	// into one `bindP` expression. Meaning that all promises will be
+	// computed sequentially. Use it to avoid writing a `bindP` chain
+	// manually.
```
Comment (dmitrybure) [inline lib/promise.flow]: document what happens when there is an error

Comment (wowofbob) [reply lib/promise.flow]: I noticed that there is `allP` function which does exactly the same as `sequenceP` but has a different implementation. It uses recursion and has optimization to keep a call stack from growing too much, while `sequenceP` uses `fold` from `array.flow`. I added `sequenceP` in order to save me some time of writing `bindP` chains. So I'm not sure which one is better and what to do about it. What do you think? @alstrup @dmitrybure

Comment (dmitrybure) [reply lib/promise.flow]: if it's exactly the same, also in error handling, I would use allP

**Comment by wowofbob** [comment]
Introducing similar to `allP` function is not necessary, closing this PR

**Pull request closed without merging**